client/src/components/IndividualReview.jsx

```diff
@@ -1,9 +1,29 @@
 import React from 'react';
 
-const IndividualReview = () => (
-  <div className="inline-block">
-    <h6>IndividualReview</h6>
+const IndividualReview = ({review}) => {
```

Comment:
I am not sure whether airbnb style guide wants us to use ( after =>. So we don't have to use a return statement. Double check would be nice.